nemo/collections/asr/metrics/bleu.py

```diff
@@ -82,7 +82,10 @@ def on_validation_epoch_end(self):
     def __init__(
         self,
         decoding: Union[AbstractCTCDecoding, AbstractRNNTDecoding, AbstractMultiTaskDecoding],
-        tokenize: Literal["none", "13a", "zh", "intl", "char"] = "13a",
+        tokenize: (
+            Literal["none", "13a", "zh", "intl", "char", "ja-mecab", "ko-mecab", "flores101", "flores200"]
+            | Dict[str, Literal["none", "13a", "zh", "intl", "char", "ja-mecab", "ko-mecab", "flores101", "flores200"]]
+        ) = "13a",
```

Comment:
Could you update it something like:
```python
TokenizeType = Union[
    Literal["none", "13a", "zh", "intl", "char", "ja-mecab", "ko-mecab", "flores101", "flores200"],
    Dict[str, Literal["none", "13a", "zh", "intl", "char", "ja-mecab", "ko-mecab", "flores101", "flores200"]]
]
```
and use it in init. Readability is not good with current way.

Reply:
How about this:

```
type SacreBLEUToken = Literal["none", "13a", "zh", "intl", "char", "ja-mecab", "ko-mecab", "flores101", "flores200"]
```
```
tokenize: Union[SacreBLEUToken, Dict[str, SacreBLEUToken]] = "13a",
```

Reply:
want to keep the dict or single tokenizer option clear in the init function

Reply:
Actually turns out each SacreBLEU metric instantiates all tokenizers in backend, see `_get_tokenizer` for new behavior.